README.md

```diff
@@ -55,6 +58,15 @@ The only thing you need to do is passing children. These children **must** have
 >
   {children}
 </ReactMotionFlip>
+
+// elements and classes specified
+<ReactMotionFlip
+  element="ul"
+  childElement="li"
```

Comment:
Reading the code, it seems that `element` and `childElement` should be renamed to `component` and `childComponent` here.